Lib/ufoLib/glifLib.py

```diff
@@ -372,7 +370,7 @@ def deleteGlyph(self, glyphName):
 	# dict-like support
 
 	def keys(self):
-		return list(self.contents.keys())
+		return self.contents.keys()
```

Comment:
this changes the API, are we sure we want to go that route?

Reply:
I did the same in defcon before. keys() isn't ordered to start with, so unless something's using it differently that as an iterator... on python 2 this will still be a list.